src/app/side-bar/list/sub-header-list/sub-header-list.component.scss

```diff
@@ -0,0 +1,39 @@
+@import 'src/assets/styles/variables';
+
+.subheader-container {
+  padding: 8px 4px;
+  border: 1px solid $blue-12;
+  width: 93rem;
```

Comment:
same here...please try to avoid using big numbers for the value

Reply:
resolved .....................Thank you